docs/source/installation.md

```diff
@@ -1,13 +1,30 @@
 (installation)=
 # Installation
 
-To install PyMC, select the operating system where you want to perform the installation.
+We recommend using [Anaconda](https://www.anaconda.com/) (or [Miniforge](https://github.com/conda-forge/miniforge)) to install Python on your local machine, which allows for packages to be installed using its `conda` utility.
 
-## Linux
-[Linux installation guide](https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Linux))
+Once you have installed one of the above, PyMC can be installed into a new conda environment as follows:
 
-## MacOS
-[MacOS installation guide](https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(MacOS))
+```console
+conda create -c conda-forge -n pymc_env pymc
+conda activate pymc_env
+```
+If you like, replace the name `pymc_env` with whatever environment name you prefer.
+
+
+## JAX sampling
+
+If you wish to enable sampling using the JAX backend via NumPyro (experimental), the following should also be installed:
+
+```console
+pip install jax jaxlib numpyro
```

Comment:
I find it a bit confusing that jax and jaxlib are included here but not for blackjax. Both numpyro and blackjax depend on jax and jaxlib and will install both packages when installing them. I'd suggest doing the same in both cases (either with or without jax+jaxlib, that part I don't really care)

Reply:
Actually, its the reverse! Numpyro will install jax and jaxlib by default, but blackjax will not. I will switch them.